initfiles/componentfiles/configschema/xsd/esp.xsd

```diff
@@ -95,7 +95,7 @@
                                         </xs:restriction>
                                     </xs:simpleType>
                                 </xs:attribute>
-                                <xs:attribute name="ldapServer" type="xs:string" use="optional" hpcc:displayName="LDAP Server" hpcc:requiredIf=".[@method=('ldap','ldaps')]"
+                                <xs:attribute name="ldapServer" type="xs:string" use="optional" hpcc:displayName="LDAP Server" hpcc:requiredIf="../EspBinding[@method=('ldap','ldaps')]"
```

Comment:
"Authentication/@ldapServer" and "Authentication/@method" seem to be at the same level, and we might not need to go up and down to EspBinding.

Reply:
Well, it was correct in the previous revision. Not sure why I changed it. Corrected now.